### PR TITLE
Adds workload to app state, advisor card to consume/use

### PR DESCRIPTION
**Pull request by AllenBW**
fixes https://issues.redhat.com/browse/ADVISOR-1364

#### as yah can see, new attribute everywhere, but wont do much till advisor prod api supports it (also wont do much if the attribute doesn't exist)
Screen Shot 2020-09-23 at 9 36 49 AM

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ const App = (props) => {
         insights.chrome?.globalFilterScope?.('insights');
         if (insights.chrome?.globalFilterScope) {
             insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
+                data?.Workloads ? dispatch(setWorkloads(data.Workloads)) : dispatch(setWorkloads(undefined));
```
Comment (karelhala) [inline src/App.js:38]: Can be simplified with
```suggestion
                dispatch(setWorkloads(data?.Workloads))
```

Comment (AllenBW) [reply src/App.js:38]: oh slick yeah i'll change that

**Review by karelhala — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -35,6 +35,7 @@ const App = (props) => {
         insights.chrome?.globalFilterScope?.('insights');
         if (insights.chrome?.globalFilterScope) {
             insights.chrome.on('GLOBAL_FILTER_UPDATE', ({ data }) => {
+                dispatch(setWorkloads(data?.Workloads));
                 const selectedTags = insights.chrome?.mapGlobalFilter?.(data)?.filter(item => !item.includes('Workloads')) || undefined;
                 dispatch(setSelectedTags(selectedTags));
```
Comment (rvsia) [inline src/App.js:40]: what about to join these two dispatches into one?

**Review by rvsia — DISMISSED** [review]
Just one small suggestion. Otherwise it looks good! :+1:

File: src/App.js (hunk 2 above)
Comment (AllenBW) [reply src/App.js:40]: OOOOO how do we do that? 🤔 🥇

Comment (rvsia) [reply src/App.js:40]: I would just create a new action, that will combine both actions (so there will be only one re-rerender)

```jsx
dispatch(globalFilterChange(data?.workloads, selectedTags))
```

Comment (AllenBW) [reply src/App.js:40]: OOOOO i see `batch`

**Review by rvsia — APPROVED** [review]

**Review by karelhala — APPROVED** [review]

**Pull request merged**